reference-architecture/aws-ansible/playbooks/roles/rhsm-repos/tasks/main.yaml

```diff
@@ -8,3 +8,12 @@
       - rhel-7-server-rpms
       - rhel-7-server-extras-rpms
       - rhel-7-server-ose-3.3-rpms
+
+- name: remove rhui repos
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+      - /etc/yum.repos.d/redhat-rhui-client-config.repo
+      - /etc/yum.repos.d/redhat-rhui.repo
+      - /etc/yum.repos.d/rhui-load-balancers.conf
```

Comment:
Would it be better to disable these using yum-config-manger instead?

You could also leverage the rhui repos for base/extra packages (would need to enable rhui extras), and only use rhsm repos for the ose packages.